pennylane/math/utils.py

```diff
@@ -169,7 +172,66 @@ def convert_like(tensor1, tensor2):
     return np.asarray(tensor1, like=interface)
 
 
-def get_interface(tensor):
+def get_interface(*values):
+    """Determines the correct framework to dispatch to given a tensor or a
+    sequence of tensor-like objects.
+
+    Args:
+        values (tensor_like, Sequence[tensor_like]): a sequence of tensor like objects
+
+    Returns:
+        str: the name of the interface
+
+    To determine the framework to dispatch to, the following rules
+    are applied:
+
+    * Tensors that are incompatible (such as Torch and TensorFlow tensors)

```

Comment:
Probably should add Jax to the list of things that are incompatible.

Reply:
Added mention to Jax in the docstring + added tests with jax interface